hw/hw1/ex_04.py

```diff
@@ -0,0 +1,29 @@
+# Задача 8: Требуется определить, можно ли от шоколадки
+#  размером n × m долек отломить k долек, 
+#  если разрешается сделать один разлом по
+#   прямой между дольками (то есть разломить 
+#   шоколадку на два прямоугольника).
+
+# *Пример:*
+
+# 3 2 4 -> yes
+# 3 2 1 -> no
+
+# 111
+# 111
+
+print("введите колво долек в длину: ")
+m = int(input())
+print("введите колво долек в ширину: ")
+n = int(input())
+print("скко долек хочешь отломить по ровной линии поделив шклдку на прямоугольники: ")
+k = int(input())
+
+if (k%m==0 or k%n==0):
```

Comment:
```suggestion
if k % m == 0 or k % n == 0:
```
Я убрал скобки, так как они в данном случае не нужны. Однако, ваше условие не полное.